File: SaleTech.Entity/Customer.cs
using System;

namespace TechSale.Entity
{
    public class Customer
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Phone { get; set; }
        public int GenderId { get; set; }
        public Gender Gender { get; set; }
        public int? CityId { get; set; }
        public City City { get; set; }
        public int? RegionId { get; set; }
        public Region Region { get; set; }
        public DateTime? LastPurchase { get; set; }
        public int? ClassificationId { get; set; }
        public Classification Classification { get; set; }
        public int? UserId { get; set; }
        public UserSys UserSys { get; set; }
    }
}

File: SaleTech-APP/src/app/_models/Customer.ts
import { Gender } from './Gender';
import { Region } from './Region';
import { City } from './City';
import { Classification } from './Classification';
import { UserSys } from './UserSys';

export class Customer {
     id: number;
     name: string ;
     phone: string ;
     genderId: number;
     gender: Gender;
     cityId: number;
     city: City;
     regionId: number;
     region: Region;
     lastPurchase: Date;
     classificationId: number;
     classification: Classification;
     userSysId: number;
     userSys: UserSys;
}
